packages/sliderInput/style.ts

```diff
@@ -0,0 +1,181 @@
+    border-width: ${sliderRoundBorder}px;
+    border-color: ${themeBgDisabled};
+    background-color: ${themeBgDisabled};
+
+    &:before {
+      border: solid;
+      border-color: ${themeBgDisabled};
+      border-width: ${sliderRoundBorder}px;
+    }
+  `,
+  "standard-focus": css`
+    border: solid;
+    border-width: ${sliderRoundBorder}px;
+    border-color: ${themeBrandPrimary};
+
+    &:before {
+      border: solid;
+      border-color: ${themeBrandPrimary};
+      border-width: ${sliderRoundBorder}px;
+    }
+  `,
+  "standard-active": css`
+    border: solid;
+    border-color: ${themeBrandPrimary};
+    border-width: ${sliderRoundBorder}px;
+    background-color: ${themeBrandPrimary};
+
+    &:before {
+      border: solid;
+      border-color: ${themeBrandPrimary};
+      border-width: ${sliderRoundBorder}px;
+      transform: translateX(${sliderInputHeight + sliderRoundBorder}px);
+    }
+  `,
+  "error-active": css`
+    border: solid;
+    border-color: ${themeBrandPrimary};
+    border-width: ${sliderRoundBorder}px;
+    background-color: ${themeBrandPrimary};
+
+    &:before {
+      border: solid;
+      border-color: ${themeBrandPrimary};
+      border-width: ${sliderRoundBorder}px;
+      transform: translateX(${sliderInputHeight + sliderRoundBorder}px);
+    }
+  `,
+  "success-active": css`
+    border: solid;
+    border-color: ${themeBrandPrimary};
+    border-width: ${sliderRoundBorder}px;
+    background-color: ${themeBrandPrimary};
+
+    &:before {
+      border: solid;
+      border-color: ${themeBrandPrimary};
+      border-width: ${sliderRoundBorder}px;
+      transform: translateX(${sliderInputHeight + sliderRoundBorder}px);
+    }
+  `,
+  "error-focus": css`
+    border: solid;
+    border-width: ${sliderRoundBorder}px;
+    border-color: ${themeError};
+
+    &:before {
+      border: solid;
+      border-width: ${sliderRoundBorder}px;
+      border-color: ${themeError};
+    }
+  `,
+  "success-focus": css`
+    border: solid;
+    border-width: ${sliderRoundBorder}px;
+    border-color: ${themeSuccess};
+
+    &:before {
+      border: solid;
+      border-width: ${sliderRoundBorder}px;
+      border-color: ${themeSuccess};
+    }
+  `,
+  "disabled-active": css`
+    border: solid;
+    border-width: ${sliderRoundBorder}px;
+    border-color: ${themeTextColorDisabled};
+    background-color: ${themeTextColorDisabled};
+
+    &:before {
+      border: solid;
+      border-width: ${sliderRoundBorder}px;
+      border-color: ${themeTextColorDisabled};
+    }
+  `,
+  "focus-active": css``
+};
+
+export const sliderContainer = css`
+  height: ${sliderInputHeight}px;
+  position: relative;
+  width: ${sliderInputWidth}px;
+`;
+
+export const slider = css`
+  background-color: ${themeBgPrimary};
+  border: solid;
+  border-width: ${sliderRoundBorder}px;
+  border-color: ${themeBorder};
+  bottom: 0;
+  cursor: pointer;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  transition: 0.4s;
```

Comment:
This is totally subjective so no action needed, but this feels a bit long to me? It looks like we commonly use 150-250ms transitions in other places

Reply:
This was taken directly from the CodePen I modeled this from so will change to be more inline with other components.

Reply:
@natmegs good eye

Reply:
I can't review the interactive aspects but thanks @natmegs for reviewing such. trust your eye